File: Web/kashish12_kashishalikhan_2125cse1133_02/GettingStarted/inverseofanumber.cpp
#include <iostream>
#include <math.h>
using namespace std;

int main(){
   int n, r, num = 0, c = 0, a;
    cin>>n;
    a = n;
    while(a > 0)
    {
        r = a % 10;
        c++;
        num += pow(10, r - 1) * c;
        a /= 10;
    }
    cout<<num<<endl;
   return 0; 
    
}

File: Android/sukritoberoi_SukritKaurOberoi_2125cse1065_2/GettingStarted/InverseOfANumber.java
import java.util.*;

public class Main{

public static void main(String[] args) {
  // write your code here  
  long n,n1;
  long d=0,f,p;
  double k;
  Scanner sc=new Scanner(System.in);
  n=sc.nextLong();
  n1=n;
  while(n1!=0)
  {
      d++;
      n1=n1/10;
  }
  for(p=1;p<=d;p++)
  {
      f=n%10;
      k=p*Math.pow(10,f);
      n1=n1+Math.round(k);
      n=n/10;
  }
  n1=n1/10;
  
  System.out.println(n1);
 }
}